### ISSUE TITLE
Relatório de todos os sites de outro domínio.

### ISSUE THREAD

**Issue by LuizzFernando26** [Wanderson304/Script-Zeus-365#6]
A função para gerar relatório de todos os sites do Sharepoint está retornando sites de outro Domínio, mesmo eu colocando o meu domínio corretamente, pois em outras funções retornou normalmente os dados corretos relacionados ao meu Domínio.

![image](https://github.com/user-attachments/assets/093fbb3b-680e-4620-82d5-78b41a786851)

**Comment by Wanderson304** [comment]
Olá Hellkkk, quando baixou o Zeus 365 ?

Ontem lancei uma nova verão a 1.2.0

Por favor, baixe novamente. Faça o seguinte:

Exclua a pasta Zeus-365 do seu Disco C:/ em seguida baixe o Zeus-35  [AQUI](https://github.com/Wanderson304/Script-Zeus-365/blob/main/Script-Zeus.rar)

**Comment by Wanderson304** [comment]
Importante, esse é um template de exemplo. Você pode executar o relatório usando seu Domínio para ele gerar o dados do seu domínio.

**Comment by LuizzFernando26** [comment]
> Olá Hellkkk, quando baixou o Zeus 365 ?
> 
> Ontem lancei uma nova verão a 1.2.0
> 
> Por favor, baixe novamente. Faça o seguinte:
> 
> Exclua a pasta Zeus-365 do seu Disco C:/ em seguida baixe o Zeus-35 [AQUI](https://github.com/Wanderson304/Script-Zeus-365/blob/main/Script-Zeus.rar)

Eu já tinha baixado a versão 1.2.0, mesmo assim exclui e baixei novamente, e continua acontecendo, mesmo colocando meu Domínio, ele mostra apenas esse template, sendo que funcionou normalmente em outras funções.

![image](https://github.com/user-attachments/assets/8d8c6f16-75c8-41bf-b600-f9f95b781116)

![image](https://github.com/user-attachments/assets/07b2855c-0784-43e2-8dbe-28fcb5913ded)

**Comment by Wanderson304** [comment]
Agora vi o erro.

Você não precisa colocar  pradum.com.br, é apenas pradum sem .com ou .br.

![image](https://github.com/user-attachments/assets/a40414d0-1393-40eb-b3e0-ac2fc158d37a)

**Comment by LuizzFernando26** [comment]
> Agora vi o erro.
> 
> Você não precisa colocar pradum.com.br, é apenas pradum sem .com ou .br.
> 
> ![image](https://private-user-images.githubusercontent.com/24323230/399311989-a40414d0-1393-40eb-b3e0-ac2fc158d37a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1NzMyNzIsIm5iZiI6MTczNTU3Mjk3MiwicGF0aCI6Ii8yNDMyMzIzMC8zOTkzMTE5ODktYTQwNDE0ZDAtMTM5My00MGViLWIzZTAtYWMyZmMxNThkMzdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMwVDE1MzYxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkYThiOWE2OTRlMDU2YmYzMjJmMzhlN2ZlZDNlZWQ4ODQyODc4YjlmOWE5ZmE5YTY2MDdjZmQ0YmI2MGE1MTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xeGOXs_EoA9GOLXAQMRyX0VtZztPWCYgIHDQ21wS0JI)

Tentei novamente e não foi.

![image](https://github.com/user-attachments/assets/b8e1c1b6-d263-4b23-b946-daa0c3c92d38)

Mesmo com o domínio correto, ainda retorna o template de exemplo.

![image](https://github.com/user-attachments/assets/a4b98291-8e47-42eb-a885-7c9b279be833)

**Comment by Wanderson304** [comment]
Olá, não indentificamos problema nessa versão, porém ja temos ums nova versão. A v1.2.1, por favor, instala essa nova versão.

**Comment by LuizzFernando26** [comment]
> Olá, não indentificamos problema nessa versão, porém ja temos ums nova versão. A v1.2.1, por favor, instala essa nova versão.

Bom dia, 

Baixei a nova versão e verifiquei que continuo com o mesmo problema, ele retorna os dados de outro Domínio no relatório, também identifiquei que ocorre o mesmo problemas em outros relatórios:

Sharepoint - Relatórios 1,2 e 4
M365/Entra ID - Relatório 4

Porém verifiquei em todos os outros relatórios restantes, e funcionou normalmente, retornando os dados no relatório referente ao meu Domínio. Segue em anexo um possível erro que possa estar ocasionando, embora tenha acontecido somente no relatório 4 do Sharepoint.

![imagem (32)](https://github.com/user-attachments/assets/70d3a02d-88e5-4c3b-b64e-172a179a3129)

Já tentei executar como Administrador também, e continua da mesma forma.

**Comment by Wanderson304** [comment]
Pelo erro o modulo de conexão, pode ser permissão, esse erro sempre estar relacionando a permissão do usuário no Microsoft 365. Aqui a base de conhecimento sobre esse erro:

https://www.sharepointdiary.com/2019/06/sharepoint-online-remote-server-returned-an-error-401-unauthorized.html

**Comment by Wanderson304** [comment]
Olá, Você seguiu esse tutorial de instalação ?

https://github.com/Wanderson304/Script-Zeus-365/wiki/Instala%C3%A7%C3%A3o

**Comment by LuizzFernando26** [comment]
> Pelo erro o modulo de conexão, pode ser permissão, esse erro sempre estar relacionando a permissão do usuário no Microsoft 365. Aqui a base de conhecimento sobre esse erro:
> 
> https://www.sharepointdiary.com/2019/06/sharepoint-online-remote-server-returned-an-error-401-unauthorized.html

Não entendo, pois a conta na qual utilizo possui Administrador Global, não vejo como poderia ser um problema de permissão, sendo que nos outros funcionou normalmente.

**Comment by LuizzFernando26** [comment]
> Olá, Você seguiu esse tutorial de instalação ?
> 
> https://github.com/Wanderson304/Script-Zeus-365/wiki/Instala%C3%A7%C3%A3o

Sim, realizei da forma que está aí, realizei novamente na hora de atualizar também.

**Comment by Wanderson304** [comment]
Se quiser entra aqui na call para eu dá uma olhada rápido:

https://meet.google.com/xwe-sngd-ftd

Vou deixar a sala aberta por 10 minutos.

**Comment by Wanderson304** [comment]
Tenta essa solução aqui:

[SOLUÇÃO](https://github.com/Wanderson304/Script-Zeus-365/issues/9)

**Comment by LuizzFernando26** [comment]
.> Tenta essa solução aqui:
> 
> [SOLUÇÃO](https://github.com/Wanderson304/Script-Zeus-365/issues/9)

Bom dia, tudo bem?

Realizei os passos na solução e deu certo, os relatórios que estavam ocorrendo esse problema funcionaram como deveriam.

**Comment by Wanderson304** [comment]
@Hellkkk,

Muito bom. Vou deixar essa discussão aberta. Assim ess0a discussão pode ajudar outros colegas.
